src/docker-osx-dev

```diff
@@ -1208,6 +1236,12 @@ function handle_command {
         DOCKER_MACHINE_NAME="$2"
         shift
         ;;
+      --only-dependencies)
+        ONLY_DEPENDENCIES=true
```

Comment:
Instead of making these global, mutable variables, could you pass them as parameters to the `install` function?